kmk/firmware.py

```diff
@@ -22,9 +22,19 @@
 import kmk.types  # isort:skip
 import kmk.util  # isort:skip
 
+import busio
+import gc
+
 # Now handlers that will be used in keys later
 import kmk.handlers.layers
 import kmk.handlers.stock
+import supervisor
```

Comment:
can we sort this by the isort criteria list? at least as much as is feasible given we're hacking import order on purpose. Still, sorting within the section would be nice.

```
isort parses specified files for global level import lines (imports outside of try / except blocks, functions, etc..) and puts them all at the top of the file grouped together by the type of import:

    Future
    Python Standard Library
    Third Party
    Current Python Project
    Explicitly Local (. before import, as in: from . import x)
    Custom Separate Sections (Defined by forced_separate list in configuration file)
    Custom Sections (Defined by sections list in configuration file)

Inside of each section the imports are sorted alphabetically. isort automatically removes duplicate python imports, and wraps long from imports to the specified line length (defaults to 79).
```

Reply:
actually it looks like the import order hackery got screwed up in this commit. I'm surprised your keyboard even boots......

Reply:
That was actually a change that was not done intentionally and was done my make fix-isort, OR githubs auto merge when I rebased. Manually changed back.